specification/cognitiveservices/data-plane/QnAMaker/stable/v4.0/QnAMaker.json

```diff
@@ -18,7 +18,7 @@
     }
   ],
   "x-ms-parameterized-host": {
-    "hostTemplate": "{Endpoint}/qnamaker/v4.0",
+    "hostTemplate": "https://{Endpoint}/qnamaker/v4.0",
```

Comment:
We want to keep the {Endpoint} configurable as cognitive services are moving to a new form of endpoint 'cognitiveservices.azure.com' instead of 'api.cognitive.microsoft.com'